pyccl/tracers.py

```diff
@@ -311,9 +308,9 @@ def get_transfer(self, lk, a):
             a (float or array_like): values of the scale factor.
 
         Returns:
-            array_like: list of transfer functions for each tracer. \
-                The shape will be `(n_tracer, lk.size, a.size)`, where \
-                `n_tracer` is the number of tracers. The other \
+            array_like: list of transfer functions for each tracer.
+                The shape will be `(n_tracer, lk.size, a.size)`, where
+                `n_tracer` is the number of tracers. The other
                 dimensions will be squeezed if the inputs are scalars.
         """
         if not hasattr(self, '_trc'):
```

Comment:
should we not remove this the same way you did for `get_kernel` and `get_f_ell`?

Reply:
Same question for `get_bessel_derivative`

Reply:
Indeed, I removed them.